src/gui/statusbar.cpp

```diff
@@ -212,6 +227,34 @@ void StatusBar::updateDHTNodesNumber()
     }
 }
 
+void StatusBar::updateExternalAddressesLabel()
+{
+    const QString lastExternalIPv4Address = BitTorrent::Session::instance()->lastExternalIPv4Address();
+    const QString lastExternalIPv6Address = BitTorrent::Session::instance()->lastExternalIPv6Address();
+    QString addressText = tr("External IP: N/A");
+
+    const bool hasIPv4Address = !lastExternalIPv4Address.isEmpty();
+    const bool hasIPv6Address = !lastExternalIPv6Address.isEmpty();
+
+    if (hasIPv4Address || hasIPv6Address)
+    {
+        if (hasIPv4Address && hasIPv6Address)
+        {
+            addressText = tr("External IPs: %1, %2")
+                    .arg(BitTorrent::Session::instance()->lastExternalIPv4Address())
+                    .arg(BitTorrent::Session::instance()->lastExternalIPv6Address());
+        }
+        else
+        {
+            addressText = tr("External IP: %1%2")
```

Comment:
This will look weird if one of them is empty.

Reply:
If one of them is empty it will result in "External IP: X" without any extra whitespace.